.github/workflows/elixir.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - uses: actions/cache/restore@v3
         name: Restore Elixir Deps Cache
         env:
-          cache-name: cache-elixir-deps-${{ env.MIX_ENV }}
```

Comment:
We should not remove MIX_ENV from the cache key. The cache is stored in `_build/${ENV_NAME}`, this means that we can generate a cache entry that will have `_build/dev`, for example, but not `_build/test`. It will result in CI working slower. It also might pull cache which is irrelevant just making things slower.

Reply:
Fixed